### PR TITLE
Wow, I updated the README!

### PR DESCRIPTION
**Pull request by HurleybirdJr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # juddweb
  
- test yo
```
Comment (juddb1) [inline README.md]: No!

**Review by juddb1 — CHANGES_REQUESTED** [review]
I don't like what it says :(

File: README.md (hunk 1 above)
Comment (HurleybirdJr) [reply README.md]: Why not?

**Review by juddb1 — APPROVED** [review]

**Review by juddb1 — APPROVED** [review]

**Comment by juddb1** [comment]
Yeet

**Pull request merged**